src/SPARQLStore/QueryEngine/QueryEngine.php

```diff
@@ -63,7 +63,8 @@ class QueryEngine {
 	 * @param QueryResultFactory $queryResultFactory
 	 * @param EngineOptions|null $engineOptions
 	 */
-	public function __construct( SparqlDatabase $connection, CompoundConditionBuilder $compoundConditionBuilder, QueryResultFactory $queryResultFactory, EngineOptions $engineOptions = null ) {
+	 // @codingStandardsIgnoreStart phpcs, ignore --sniffs=Generic.Files.LineLength
```

Comment:
For most exceptions, this format is nice:

```
// @codingStandardsIgnoreStart
stuff that should be ignored here
// @codingStandardsIgnoreEnd
```

For some you cannot do this (ie unused parameters), though for most, such as line length, it's quite easy.

With how it's done now, I'll be able to see the open tag, but not the closing one, as it's at the end of an already too long line :)